portfolio/src/main/java/com/google/sps/data/CommentList.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.data;
+
+import java.util.List;
+import java.util.ArrayList;
+
+public class CommentList {
+    private List <Comment> commentList;
+
+    public CommentList() {
+        this.commentList = new ArrayList <Comment>();
+    }
+
+    public CommentList(List <Comment> commentList) {
+        this.commentList = commentList;
+    }
+
```

Comment:
I would only use List in an interface rather than a concrete class. In this case you are using ArrayList then converting it to List then casting it back to ArrayList

Reply:
I changed the it to List on the Servlet, so I don't think this is needed anymore!